release/models/macsec/openconfig-macsec.yang

```diff
@@ -17,11 +18,17 @@ module openconfig-macsec {
     "This module defines configuration and state data for
      MACsec IEEE Std 802.1AE-2018.";
 
-  oc-ext:openconfig-version "0.2.0";
+  oc-ext:openconfig-version "0.3.0";
```

Comment:
Need to bump major revision number since this is a breaking change,  ie:  (1.0.0 instead of 0.3.0)

Reply:
Bumped up the version to 1.0.0 at couple of places and updated the revision date as well.